ospfd/ospf_vty.c

```diff
@@ -983,7 +983,8 @@ DEFUN (ospf_area_vlink,
        "Key ID\n"
        "Use MD5 algorithm\n"
        "The OSPF password (key)\n"
-       "Authentication password (key)")
+       "Authentication password (key)\n"
```

Comment:
There are a few macros above this function that seem to exist only to be used here I think, like: VLINK_HELPSTR_AUTH_MD5 and VLINK_HELPSTR_AUTH_SIMPLE , should we use those?